File: examples/go/mono/libs/math/math.go
package mathlib
func Add(a,b int) int { return a+b }
func Sub(a,b int) int { return a-b }
func Mul(a,b int) int { return a*b }
func Div(a,b int) int { return a/b }
func Mod(a,b int) int { return a%b }
func Square(a int) int { return a*a }
func Cube(a int) int { return a*a*a }
func Max(a,b int) int { if a>b {return a}; return b }
func Min(a,b int) int { if a<b {return a}; return b }
func Abs(a int) int { if a<0 {return -a}; return a }

File: examples/kotlin/mono/libs/math/src/main/kotlin/ayni/math/Math.kt
package ayni.math

fun add(left: Int, right: Int): Int = left + right
fun subtract(left: Int, right: Int): Int = left - right
fun multiply(left: Int, right: Int): Int = left * right
fun divide(left: Int, right: Int): Int = left / right
fun square(value: Int): Int = value * value
fun cube(value: Int): Int = value * value * value
fun max(left: Int, right: Int): Int = if (left > right) left else right
fun min(left: Int, right: Int): Int = if (left < right) left else right
fun increment(value: Int): Int = value + 1
fun decrement(value: Int): Int = value - 1
